### DISCUSSION TITLE
Remove crud

### DISCUSSION THREAD

**Discussion by dmatis2** [Laravel-Backpack/community-forum#488] [Q&A (Help)] [answered]
How can I remove CRUD from Laravel Backpack? I've run `php artisan backpack:crud <model>` but now I want to undo it, what I need to do ? I cannot find any command for it. Thanks

**Comment by christmex** [comment, marked as answer]
Hiii, here are the manual steps

1. delete the line of route, located at ``` routes/backpack/custom.php ```
2. delete the controller, located at ``` app/Http/Controllers/Admin/EntityCrudController.php  ```
3. delete the line from the sidebar, located at ``` resources/views/vendor/backpack/base/inc/sidebar_content.blade.php ```
4. delete the model and the migration *just in case you want it too

then, perform ``` php artisan optimize:clear ```


let me know if you have any issues, thx

**Reply by dmatis2** [reply]
thank you
